tensorboard/uploader/exporter_test.py

```diff
@@ -234,26 +234,28 @@ def test_stream_experiment_data_passes_through_unexpected_exception(self):
     #   1. stream_experiments will say there is one experiment_id.
     #   2. stream_experiment_data will throw an internal error.
     mock_api_client = self._create_mock_api_client()
-    experiment_id="123"
+    experiment_id = "123"
 
     def stream_experiments(request, **kwargs):
       del request  # unused
       yield export_service_pb2.StreamExperimentsResponse(
-          experiment_ids=[experiment_id])
+          experiment_ids = [experiment_id])
```

Comment:
nit: No spaces around kwargs. The full whitespace rules are here:
<https://google.github.io/styleguide/pyguide.html#36-whitespace>